internal/rest/dequeue_nodes.go

```diff
@@ -631,7 +631,7 @@ func (rest *RestOperations) PopulateOneCache(rest_op *utils.RestOp, aviObjKey av
 			rest.AviVsVipCacheAdd(rest_op, aviObjKey, key)
 		}
 
-	} else {
+	} else if rest_op.Method == utils.RestDelete {
```

Comment:
I think we can add a error check here too 
else if (rest_op.Err == nil) && (rest_op.Method == utils.RestDelete)

Reply:
Addressed. Thanks.